imitation/scripts/config/data_collect.py

```diff
@@ -8,10 +8,20 @@
 
 @data_collect_ex.config
 def data_collect_defaults():
-    total_timesteps = int(1e6)
-    num_vec = 8
+    env_name = "FILL ME"  # The gym.Env name
```

Comment:
It's nice to be able to have any script run by default with no config, so perhaps set this to e.g. Pendulum or CartPole?